egs/librispeech/ASR/pruned_stateless_emformer_rnnt2/emformer.py

```diff
@@ -296,7 +297,7 @@ def get_init_state(self, device: torch.device) -> List[List[torch.Tensor]]:
           Return the initial state of each layer. NOTE: the returned
           tensors are on the given device. `len(ans) == num_emformer_layers`.
         """
-        if len(self._init_state) > 0:
+        if self._has_init_state:
```

Comment:
We cannot use `len(self._init_state)` here. See https://github.com/k2-fsa/icefall/pull/401#discussion_r890071489